CommonTools/CandAlgos/plugins/CandPtrProjector.cc

```diff
@@ -18,12 +19,14 @@ class CandPtrProjector : public edm::global::EDProducer<> {
 private:
   edm::EDGetTokenT<edm::View<reco::Candidate>> candSrcToken_;
   edm::EDGetTokenT<edm::View<reco::Candidate>> vetoSrcToken_;
+  bool useDeltaRforFootprint_;
 };
 
 CandPtrProjector::CandPtrProjector(edm::ParameterSet const& iConfig):
   candSrcToken_{consumes<edm::View<reco::Candidate>>(iConfig.getParameter<edm::InputTag>("src"))},
   vetoSrcToken_{consumes<edm::View<reco::Candidate>>(iConfig.getParameter<edm::InputTag>("veto"))}
 {
+  useDeltaRforFootprint_ = iConfig.exists("useDeltaRforFootprint") ? iConfig.getParameter<bool>("useDeltaRforFootprint") : false;
```

Comment:
looks like fillDescriptions is in order

Reply:
even though a fillDescriptions would be nice in general, my comment was triggered mostly by the use of ::exists. 
It seems like in this case the new parameter can just be added as required and inserted in existing .py files with an appropriate default value.